README.md

```diff
@@ -162,7 +162,7 @@ The IOS procedure is similar to the macOS one:
   $ git submodule init
   $ git submodule update --remote
 1. Install python3 dependencies:
-  $ pip install glean_parser
+  $ pip3 install glean_parser
```

Comment:
Added this in here because I stumbled upon it while setting um a fresh clone.
I can put it in its own PR if wanted.